### PR TITLE
Clean up a11y issues

### PR DESCRIPTION
**Pull request by cherewaty**
Fixes the easiest wins:
- Color contrast of text in results lists
- Add readable test to icon buttons

Bonus: fixes search filters pane sizing behavior.

### CODE REVIEW

**Comment by wwilsman** [comment]
RE the chrome failure:

Looks like it can't find this selector `[class^="feedbackError--"]`

There's a possibility that the class wasn't first in it's list. `^=` checks the beginning of the attribute. So it would match `class="feedbackError--gH76k blah"` but not `class="blah feedbackError--gH76k"`

We might want to change that to `[class*="feedbackError--"]` instead. But that could possibly match `class="someother-feedbackError--something"`.

I think the safest bet is to use the generated class names by importing the style like we're doing in the [toast page object](https://github.com/folio-org/ui-eholdings/blob/master/tests/pages/toast.js).

**Comment by cherewaty** [comment]
Flaky test fix attempt: https://github.com/folio-org/ui-eholdings/pull/333

File: src/components/customer-resource-show.js [hunk 1]
```diff
@@ -136,6 +136,7 @@ export default class CustomerResourceShow extends Component {
             <PaneMenu>
               <IconButton
                 icon="edit"
+                title={`Edit ${model.name}`}
```
Comment (Robdel12) [inline src/components/customer-resource-show.js]: So I know we have no control over this right now but `title` is not for a11y. Here's a great resource as to why: https://a11yproject.com/posts/title-attributes/

But the TL;DR screen readers treat `title` differently. If you want to label something use `aria-label`

File: src/components/package-list-item/package-list-item.js [hunk 2]
```diff
@@ -32,9 +32,12 @@ export default function PackageListItem({
         'is-selected': active
       })}
     >
-      <h5 data-test-eholdings-package-list-item-name>
+      <div
```
Comment (Robdel12) [inline src/components/package-list-item/package-list-item.js]: If not an `h5` is this better semantically as a paragraph tag?

Comment (cherewaty) [reply src/components/package-list-item/package-list-item.js]: I took this commit back out. The headings are more semantically correct, but we need them to be an h3 in one instance and an h4 in another. I'll revisit that later.

File: src/components/customer-resource-show.js (hunk 1 above)
Comment (cherewaty) [reply src/components/customer-resource-show.js]: Lucky for us... `IconButton` has an `ariaLabel` prop. Changed.

Comment (Robdel12) [reply src/components/customer-resource-show.js]: Oh good! I'd like to deprecate the `title` prop entirely. It should pretty much never be used.

**Pull request merged**